src/repositories/channel.js

```diff
@@ -7,7 +7,7 @@ module.exports = [
         const get = async (opt) => {
             const channel;
             if(opt.hasOwnProperty('lean')){
```

Comment:
```suggestion
           const channel = channelModel.findOne(opt.query).lean(opts.lean);      
```

You already setting it to true. If the value is false or not set, the query will not be executed as a lean.